gvm/protocols/gmpv7/gmpv7.py

```diff
@@ -3860,16 +3919,19 @@ def get_scanner(self, scanner_id: str) -> Any:
 
         Arguments:
             scanner_id: UUID of an existing scanner
+            details:  Whether to include extra details like tasks using this
+                scanner
```

Comment:
The implementation for details is missing

Reply:
I first added these parameters, but then noticed:
```
# for single entity always request all details
```
So these parameters are not necessary ... (I hope ...)